ABBREV.HELPREXX

```diff
@@ -14,5 +14,5 @@ Examples:
     ABBREV ('PRINT' , 'Pri')  == 0
     ABBREV('PRINT', 'PRI' ,4) == 0
     ABBREV('PRINT', 'PRY')    == 0
-    ABBREV ( 'PRINT' , ' ' )  == 1
```

Comment:
This should actually have been `ABBREV('PRINT', '') == 1`, because it's a copy-and-paste from the examples in "_The Rexx Language_", 2nd ed.  So my error was in putting a blank inside the single-quotes in the second operand.